README.md

```diff
@@ -5,7 +5,8 @@
 ### Building:
 
 ```bash
-git clone https://github.com/opencontainers/runc
+git clone https://github.com/opencontainers/runc $GOPATH/src/github.com/opencontainers/runc
```

Comment:
I guess it depends do we expect the audience of the README to potentially be new to go. Note http://golang.org/doc/code.html uses $GOPATH 

Users who have setup a more complex GOPATH probably know what they are doing and can adjust the command. So I think it's fine to have the simple case in the README Alternately one could use 

```
go get -d github.com/opencontainers/runc
```